src/gpu/generic/sycl/ref_convolution.hpp

```diff
@@ -203,7 +205,8 @@ struct ref_convolution_bwd_weights_t : public gpu::generic::sycl::primitive_t {
                     && attr()->has_default_values(sm::scales_runtime
                             | sm::zero_points_runtime | sm::sum_dt)
                     && IMPLICATION(!attr()->scales_.has_default_values(),
-                            attr_scales_ok());
+                            attr_scales_ok())
```

Comment:
(not related to this PR) there should be no scaling for bwd pass.

Reply:
fixed, along with post ops